ccdproc/combiner.py

```diff
@@ -317,3 +318,206 @@ def average_combine(self, scale_func=None, scale_to=1.0):
+
+        Parameters below are valid only when minmax_clip is set to True.
+
+        minmax_clip_min: None, float
+             All pixels with values below minmax_clip_min will be masked.
+        minmax_clip_max: None or float
+             All pixels with values above minmax_clip_max will be masked.
+
+    sigma_clip : Boolean (default False)
+        Set to True if you want to reject pixels which have deviations greater than those
+        set by the threshold values.   The algorithm will first calculated
+        a baseline value using the function specified in func and deviation
+        based on sigma_clip_dev_func and the input data array.   Any pixel with a
+        deviation from the baseline value greater than that set by
+        sigma_clip_high_thresh or lower than that set by sigma_clip_low_thresh will be rejected.
+
+        Parameters below are valid only when sigma_clip is set to True.
+
+        sigma_clip_low_thresh : positive float or None
+            Threshold for rejecting pixels that deviate below the baseline
+            value.  If negative value, then will be convert to a positive
+            value.   If None, no rejection will be done based on sigma_clip_low_thresh.
+
+        sigma_clip_high_thresh : positive float or None
+            Threshold for rejecting pixels that deviate above the baseline
+            value. If None, no rejection will be done based on sigma_clip_high_thresh.
+
+        sigma_clip_func : function
+            Function for calculating the baseline values (i.e. mean or median).
+            This should be a function that can handle
+            numpy.ma.core.MaskedArray objects.
+
+        sigma_clip_dev_func : function
+            Function for calculating the deviation from the baseline value
+            (i.e. std).  This should be a function that can handle
+            numpy.ma.core.MaskedArray objects.
+    
+    **ccdkwargs: Other keyword arguments for CCD Object's fits reader.
+
+    Returns
+    -------
+    combined_image: `~ccdproc.CCDData`
+        CCDData object based on the combined input of CCDData objects.
+
+    """
+    if not isinstance(img_list,list):
+        # If not a list, check wheter it is a string of filenames seperated by comma
+        if isinstance(img_list,str) and (',' in img_list):
+            img_list = img_list.split(',')
+        else:
+            raise ValueError("Unrecognised input for list of images to combine.")
+
+    # Select Combine function to call in Combiner
+    if method == 'average':
+        combine_function = 'average_combine'
+    elif method == 'median':
+        combine_function = 'median_combine'
+    else:
+        raise ValueError("Unrecognised combine method : {0}".format(method))
+    
+    
+    # First we create a dummy output CCDObject from first image for storing output       
+    if isinstance(img_list[0],CCDData):
+        ccd_dummy = img_list[0].copy()
+    else:
```

Comment:
I think this could just be changed to ccd.   It is actually quite important and ccd_dummy might imply this object could be ignored.